src/kirby/components/list/list.event.ts

```diff
@@ -0,0 +1,8 @@
+export interface LoadOnDemandEvent {
+  complete: (disableLoadOnDemand?: boolean) => void;
+}
+
+export interface LoadOnDemandEventData {
+  object: any;
```

Comment:
```suggestion
  target: any;
```

Reply:
This `LoadOnDemandEventData` interface is ment as a wrapper for {N}'s `LoadOnDemandListViewEventData` interface. So i can not rename the properties.

Reply:
I will add a comment saying this.